packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -34,7 +34,7 @@ Creates a new user in your Okta organization with or without credentials
 - [Create User with Password & Recovery Question](#create-user-with-password-recovery-question)
```

Comment:
```suggestion
- [Create user with password & recovery question](#create-user-with-password-recovery-question)
```

Reply:
There are some other Capitalization Issues I'd like to fix above but GH won't let me